foundations/html_css/html-foundations/commit-messages.md

```diff
@@ -0,0 +1,44 @@
+### Introduction
+
+This lesson will explain the importance of good commit messages, how to write them, when to commit and why having a history of good commits is so important!
+
+### Learning Outcomes
+
+By the end of this lesson, you should be able to:
+
+- Write a meaningful commit message
+- Understand why meaningful commit messages are important
+- Have a good understanding of when to commit
```

Comment:
```suggestion
### Lesson Overview

This section contains a general overview of topics that you will learn in this lesson.

- How to write a meaningful commit message
- Why meaningful commit messages are important
- When to commit
```

Slight change to bring this in line with the [new style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md).

Reply:
Add!